pegen/c_generator.py

```diff
@@ -29,28 +29,36 @@
 """
 EXTENSION_SUFFIX = """
 static PyObject *
-parse_file(PyObject *self, PyObject *args)
+parse_file(PyObject *self, PyObject *args, PyObject *kwds)
 {
+    static const char *keywords[] = {"file", "mode", NULL};
```

Comment:
What about naming that second keyword argument something other than `mode`? I know, that's what it's called in `pegen.c`, but in the public functions we could name it something like `public_ast` and have it be a `bool`, rather than an `int`.

Reply:
I'd rather keep `mode`, since I intend to expand this to `mode=2` which would compile the code.